docs/API.md

```diff
@@ -506,3 +507,25 @@ push.clearAllNotifications(() => {
 	console.log('error');
 });
 ```
+
+## push.clearNotification(id, successHandler, errorHandler) - Android only
+
+Tells the OS to clear the notification that corresponds to the id argument, from the Notification Center
+
+### Parameters
+
+Parameter | Type | Default | Description
+--------- | ---- | ------- | -----------
+`id` | `number` | | The ID of the notification that will be cleared. |
+`successHandler` | `Function` | | Is called when the api successfully clears the notifications.
+`errorHandler` | `Function` | | Is called when the api encounters an error when attempting to clears the notifications.
```

Comment:
[typo] `notifications` here should not be plural. There's a typo on `attempting to clears` too, but that is also wrong on the original `clearAllNotifications` doc, so we could fix them both.

Reply:
I will make these changes in the next commit.